Library/Formula/caf.rb

```diff
@@ -0,0 +1,52 @@
+class Caf < Formula
+  homepage "http://actor-framework.org"
+  head "https://github.com/actor-framework/actor-framework.git"
+
+  stable do
+    url "https://github.com/actor-framework/actor-framework/archive/0.12.0.tar.gz"
+    sha256 "294fb00508c166595d479e9af2fae28f2d370ed164cd1de079ae247acc916801"
+  end
+
+  devel do
+    url "https://github.com/actor-framework/actor-framework.git",
+      :branch => "develop"
```

Comment:
Ah. Yeah, I don't think there are many active cases of ` devel do ` pointing to a ` git `, rather than some sort of tarball. I don't know how the maintainers will feel about it doing so. It's not completely out of the ordinary though, we've accepted it in some formulae in the past. I'd probably wait and see what one of the maintainers has to say on it, to be honest.